### DISCUSSION TITLE
Only use one slider for scaling the size of gameplay, rather than a separate horizontal and vertical.

### DISCUSSION THREAD

**Discussion by Skrubbadubba** [ppy/osu#14616] [Ideas]
AFAIK the aspect ratio of the playfield (in standard) is 4:3, and it stays that way. It would be more intuitive if in the settings page where you can resize and reposition the playfield, there would only be one slider that scales that playfield. 

When dragging on the sliders, the white foreground that appears doesn't seem to represent the playfield, but rather, the entire screen, inside of which is the 4:3 playfield that you cant see. Furthermore, the size of hitobjects doesnt seem to change until the horizontal scaling is below 75%, whereas they start to shrink instantly when vertical scaling is changed, making it more confusing as to what the actual size of the playfield is.

https://user-images.githubusercontent.com/75796723/131883432-02f50d0e-a3c5-46cd-a29f-edf8ab85da45.mp4

It would make more sense if the white foreground that appears when scaling only the gameplay, is the actual playfield.

Apart from making this whole thing more intuitive, this feature would make it easier to calculate how the tablet area/mouse sensitivity should be changed when the size of the playfield is changed.

**Comment by bdach** [comment]
Instead of taking away existing functionality, it would probably be a better idea to *add* something to facilitate your use case (i.e. fixing scale aspect ratio to the aspect ratio of the tablet area).

**Reply by peppy** [reply]
Yep, something similar to the tablet area configuration. Might be able to share the implementation to some extent, not sure.
